pkgs/development/python-modules/openstep-parser/default.nix

```diff
@@ -0,0 +1,65 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  pythonOlder,
+  pytestCheckHook,
+  setuptools,
+  git,
+}:
+
+buildPythonPackage rec {
+  pname = "openstep-parser";
+  version = "2.0.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "kronenthaler";
+    repo = "openstep-parser";
+    rev = "refs/tags/${version}";
+    hash = "sha256-saRf8vm1triAaJ+iZENtreb/QFUa3OICHh/7ImXA2yo=";
+  };
+
+  build-system = [
+    setuptools
+    git
+  ];
+
+  preBuild = ''
+    echo "
+    diff --git a/setup.py b/setup.py
+    index c0caf7e..1938eb7 100644
+    --- a/setup.py
+    +++ b/setup.py
+    @@ -23,7 +23,7 @@ setup(
+         name='openstep_parser',
+         author='Ignacio Calderon',
+         description='OpenStep plist reader into python objects',
+    -    long_description=long_description,
+    +    long_description=open(\"README.md\").read(),
+         url=\"http://github.com/kronenthaler/openstep-parser\",
+         version=find_version(\"openstep_parser\", \"__init__.py\"),
+         license='BSD License',
+    " | git apply
+  '';
```

Comment:
This should be in a separate file and included in the `patches` list. (A comment with a link to the PR would be good too.)

If we’re going to ship one patch we might as well ship your https://github.com/kronenthaler/openstep-parser/pull/30 too so that we can check it works with the tests?

Reply:
added the patches and reenable all the tests and working fine now

Reply:
they send a patch fix, removed the patch